hawkular-nest/hawkular-nest-itest/pom.xml

```diff
@@ -290,23 +294,34 @@
       </plugin>
 
       <plugin>
-        <artifactId>maven-resources-plugin</artifactId>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
         <executions>
           <execution>
-            <id>copy-resources</id>
+            <id>create-itest-users</id>
```

Comment:
Why do you have a random password for the test user? If you were to use a static password, you wouldn't need to run this, right? You'd just place the appropriate `application-users.properties` and `application-roles.properties` on the WF to be tested.